### ISSUE TITLE
Super+q shortcut override

### ISSUE THREAD

**Issue by prdanelli** [adaxi/audio-output-switcher#16]
The extension seems to use super+q to change the audio device, but this is the only way to close applications. I've tried to disable / remove the extension and the shortcut doesn't get reset.

**Comment by adaxi** [comment]
If you disable the extension should not be taken into account anymore. If disabling the application does not need to do the trick, try logging out and logging back in.

Perhaps you should also check if you did not remove the shortcut in the gnome settings, under Devices > Keyboard.

You can disable the code see: #11

**Comment by prdanelli** [comment]
The problem is that the shortcut is not reset when the extension is removed. I have tried restarting my machine numerous times, searched in settings / shortcuts, inside of the dconf settings and inside of tweaks, the option to edit the shortcut is not available and therefore the shortcut cannot be removed.

**Comment by prdanelli** [comment]
_For anyone coming to this with the same issue, here is how I got the shortcut back._

The issue I had was that the extension shortcut is retained even after the extension is disabled and removed, which is annoying to say the least. 

To get round this, I edited the `extension.js` here: 

```~/.local/share/gnome-shell/extensions/audio-output-switcher@anduchs/extension.js```

Removing lines 109-128. 

Then I was able to restart my machine to make Gnome pick up the new settings. I was then able to remove the extension and get super+q back as a useable shortcut.
